buildSrc/src/main/java/Dependencies.kt

```diff
@@ -14,9 +14,10 @@ private object Versions {
     const val google_material = "1.0.0"
 
     const val android_gradle_plugin = "3.2.1"
-    const val appservices_gradle_plugin = "0.2.0"
+    const val appservices_gradle_plugin = "0.3.0"
+
+    const val mozilla_android_components = "0.40.0"
 
-    const val mozilla_android_components = "0.40.0-SNAPSHOT"
```

Comment:
@thomcc @grigoryk this probably landed here by mistake? We want to keep `r-b` on snapshots :)

This is also why the build for this PR failed. There's no 0.40.0 yet.